arangod/Aql/Ast.h

```diff
@@ -265,9 +261,21 @@ class Ast {
   AstNode* createNodeTernaryOperator(AstNode const*, AstNode const*,
                                      AstNode const*);
 
+  /// @brief create an AST variable access
+  AstNode* createNodeAccess(Variable const*,
+                            std::vector<basics::AttributeName> const&);
+
   /// @brief create an AST attribute access node
   /// note that the caller must make sure that char* data remains valid!
   AstNode* createNodeAttributeAccess(AstNode const*, char const*, size_t);
+  //AstNode* createNodeAttributeAccess(AstNode const*, std::vector<std::string_view>);
+  AstNode* createNodeAttributeAccess(AstNode const*, std::vector<std::string>);
+
+  AstNode* createNodeAttributeAccess(AstNode const* node, std::vector<basics::AttributeName> const& attrs) {
```

Comment:
Any particular reason this one is inlined? Also, could we get some documentation about the parameters for this and the one above maybe?